fe/fe-core/src/main/java/org/apache/doris/analysis/DataSortInfo.java

```diff
@@ -0,0 +1,104 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+import org.apache.doris.common.io.Writable;
+import org.apache.doris.thrift.TSortType;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.Map;
+
+public class DataSortInfo implements Writable {
+    public static final String DATA_SORT_PROPERTY_PREFIX = "data_sort";
+    public static final String DATA_SORT_TYPE = "data_sort.sort_type";
+    public static final String DATA_SORT_COL_NUM = "data_sort.col_num";
+
+    private TSortType sortType;
+    private int colNum;
```

Comment:
Use Gson to serialize.  You can refer to `DropDbInfo`
And the thrift data struct `TSortType` can be convert to a new self defined enum

Reply:
done